src/containers/pages/MDAPoint/ClientUpload/tests/index.test.tsx

```diff
@@ -13,13 +12,16 @@ describe('components/ClientUpload', () => {
     shallow(<ClientUpload />);
   });
 
-  it('Matches snapshot', () => {
+  it('renders props correctly', () => {
```

Comment:
I do not think that you have adequately tested how `ClientUpload` works.

Reply:
I created an issue to add extensive tests here https://github.com/onaio/reveal-frontend/issues/889.